tests/acceptance/cancelBuilds.sh

```diff
@@ -0,0 +1,60 @@
+SCRIPT_PATH=$(dirname "$0")
+SCRIPT_PATH=$( cd "${SCRIPT_PATH}" && pwd )  # normalized and made absolute
+OWNCLOUD_ACTIVITY="owncloud/activity"
```

Comment:
We need to pass this in to the script, because we want to be able to use the same script for all the oC10 apps.
```suggestion
REPO_SLUG="$1"
```

Reply:
fixed